core/embed/sec/tropic/stm32/tropic01.c

```diff
@@ -184,12 +186,7 @@ lt_ret_t lt_port_delay(lt_handle_t *h, uint32_t ms) {
 }
 
 lt_ret_t lt_port_random_bytes(uint32_t *buff, uint16_t len) {
```

Comment:
According to the new function signature we should have:
```suggestion
lt_ret_t lt_port_random_bytes(void *buff, size_t count) {
```